src/plugins/mini/mini.c

```diff
@@ -0,0 +1,218 @@
+/**
+ * @file
+ *
+ * @brief Source for mini plugin
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ *
+ */
+
+/* -- Imports --------------------------------------------------------------------------------------------------------------------------- */
+
+#include "mini.h"
+
+#include <kdbease.h>
+#include <kdberrors.h>
+#include <kdbhelper.h>
+#include <kdblogger.h>
+#include <kdbutility.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+/* -- Functions ------------------------------------------------------------------------------------------------------------------------- */
+
+// ===========
+// = Private =
+// ===========
+
+static inline KeySet * elektraMiniContract ()
+{
+	return ksNew (30, keyNew ("system/elektra/modules/mini", KEY_VALUE, "mini plugin waits for your orders", KEY_END),
+		      keyNew ("system/elektra/modules/mini/exports", KEY_END),
+		      keyNew ("system/elektra/modules/mini/exports/get", KEY_FUNC, elektraMiniGet, KEY_END),
+		      keyNew ("system/elektra/modules/mini/exports/set", KEY_FUNC, elektraMiniSet, KEY_END),
+#include ELEKTRA_README (mini)
+		      keyNew ("system/elektra/modules/mini/infos/version", KEY_VALUE, PLUGINVERSION, KEY_END), KS_END);
+}
+
+static inline char * stripComment (char * line)
+{
+	char * current = line;
+	char * before = NULL;
+
+	/* As long as we are not the end of the string and
+	   the current character is either not a comment marker or the comment marker was escaped */
+	while (*current != '\0' && ((*current != '#' && *current != ';') || (before && *before == '\\')))
+	{
+		before = current;
+		current++;
+	}
+	*current = '\0';
+	return line;
+}
+
+static inline char * findUnescapedEquals (char * text)
```

Comment:
If the function does anything that the name does not already say, it would be better if we have it documented.

Log output and assertions also count as docu.

For private functions you obviously do not need to be as verbose as in `elektraReplace`.

> The function will return a pointer to the \0 character at the end of the string.

Please simply add this as your documentation.